### PR TITLE
Change width for input field

### PR DESCRIPTION
**Pull request by JuliaKirschenheuter**
* Part of https://github.com/nextcloud/server/issues/36965

Change `flex-direction` to prevent horizontal scrolling

## Summary

🏚️ Before | 🏡 After
---|---
image | Screenshot from 2023-10-12 14-02-03

## Checklist

- Code is [properly formatted](https://docs.nextcloud.com/server/latest/developer_manual/digging_deeper/continuous_integration.html#linting)
- [Sign-off message](https://github.com/src-d/guide/blob/master/developer-community/fix-DCO.md) is added to all commits
- [ ] Tests ([unit](https://docs.nextcloud.com/server/latest/developer_manual/app_development/tutorial.html#unit-tests), [integration](https://docs.nextcloud.com/server/latest/developer_manual/app_development/tutorial.html#integration-tests), api and/or acceptance) are included
- [x] Screenshots before/after for front-end changes
- [ ] Documentation ([manuals](https://github.com/nextcloud/documentation/) or wiki) has been updated or is not required
- [ ] [Backports requested](https://github.com/nextcloud/backportbot/#usage) where applicable (ex: critical bugfixes)

### CODE REVIEW

File: apps/user_status/src/components/ClearAtSelect.vue [hunk 1]
```diff
@@ -87,6 +87,9 @@ export default {
 
 <style lang="scss" scoped>
 .clear-at-select {
+	@media only screen and (min-width: 320px) and (max-width: 1280px) {
+		flex-direction: column;
+	}
```
Comment (szaimen) [inline apps/user_status/src/components/ClearAtSelect.vue:92]: I think media queries need to be at root and cannot be put into any class or anything...

**Review by szaimen — CHANGES_REQUESTED** [review]
This does not seem to work for me:
image

File: apps/user_status/src/components/ClearAtSelect.vue (hunk 1 above)
Comment (JuliaKirschenheuter) [reply apps/user_status/src/components/ClearAtSelect.vue]: made alternative solution

**Review by szaimen — APPROVED** [review]
Works fine now in my testing. 

However this detail still looks a bit weird, however could potentially be fixed in a follow-up:

image

**Review by Pytal — APPROVED** [review]

**Review by ShGKme — APPROVED** [review]

**Comment by ShGKme** [comment]
Cypress and Drone tests are unrelated

**Comment by susnux** [comment]
> Cypress and Drone tests are unrelated

Yes but the commits need to be squashed (fixup)

**Review by susnux — APPROVED** [review]

**Comment by ShGKme** [comment]
Cypress fail unrelated

**Pull request merged**